packages/devtools_app/lib/src/network/network_controller.dart

```diff
@@ -168,6 +171,7 @@ class NetworkController {
       // process of starting recording.
       return;
     }
+    _httpRecordingNotifier.value = true;
```

Comment:
why are we setting this here if we are calling _toggleHttpTimelineRecording (which also sets the value to true) below in the try statement?